zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -159,6 +164,9 @@ function PageWrapper(props) {
               >
                 <TranslateIcon />
                 <Select
+                  menuProps={{
```

Comment:
like you said this doesn't work here, so perhaps we should remove it from here?

Reply:
yes i kept it just in case i'm missing something but now i removed it